cmd/skywire-cli/commands/config/gen.go

```diff
@@ -420,21 +420,25 @@ var genConfigCmd = &cobra.Command{
 		var routeSetupPKs cipher.PubKeys
 		var tpSetupPKs cipher.PubKeys
 		var surveyWhitelistPKs cipher.PubKeys
+		// If nothing was fetched
 		if services.SurveyWhitelist == nil {
-			if surveywhitelistPks != "" {
-				if err := surveyWhitelistPKs.Set(surveywhitelistPks); err != nil {
-					log.Fatalf("bad key set for survey whitelist flag: %v", err)
-				}
-				services.SurveyWhitelist = surveyWhitelistPKs
-				surveyWhitelistPKs = cipher.PubKeys{}
-			}
-			if !noDefaults {
-				if err := surveyWhitelistPKs.Set(utilenv.SurveyWhitelistPKs); err != nil {
-					log.Fatalf("Failed to unmarshal survey whitelist public keys: %v", err)
-				}
-				services.SurveyWhitelist = append(services.SurveyWhitelist, surveyWhitelistPKs...)
-			}
-		}
+		  // By default
+		  if !noDefaults {
```

Comment:
A lot of these flags have inconvenient naming. `noDefault` would be better named as something like `useDefaults` with its default flag value being set to true. This would improve readability quite a bit. Nothing to be done on this PR, but feel free to create a ticket if you agree.

Reply:
All the flags are false by default, intentionally.

I agree with you it makes for an unintuitive naming scheme in some instances

it adds complexity for setting the flags to true by default, I think to set them as false from there you have to do something like --flag=false

having weird names in some instances was the better alternative